application/application.go

```diff
@@ -25,3 +27,21 @@ func checkAllBundleContentsAreApproved(contents []models.BundleContent) bool {
 	}
 	return true
 }
+
+type Bundlestore interface {
+	ListBundles(ctx context.Context, offset, limit int) ([]*models.Bundle, int, error)
+}
+
+type BundleService struct {
+	Store Bundlestore
+}
+
+func NewBundleService(store Bundlestore) *BundleService {
+	return &BundleService{
+		Store: store,
+	}
+}
+
+func (s *BundleService) ListBundles(ctx context.Context, offset, limit int) ([]*models.Bundle, int, error) {
+	return s.Store.ListBundles(ctx, offset, limit)
```

Comment:
See the above comments to use the relevant access point to the listBundles function in the data store. Also, there should probably be an error check in here to return an error to the api layer that is calling it should the request to the data store fail